data/migrations/create-sql-functions.py

```diff
@@ -16,6 +16,8 @@ def format_value(val):
         elif 'col' in val:
             if val['col'].startswith('tags->'):
                 return "tags->'%s'" % val['col'][len('tags->'):]
+            elif val['col'].startswith('$'):
+                return '"%s"' % val['col'][len('$'):]
```

Comment:
Ah, no. I'm using the `$column` syntax to denote a column which is suppressed from the input, but thinking about it now, that's unnecessary. I've added something in 4857b8c which hopefully makes the usage clearer and removes some unnecessary code branches.